mailet/base/src/main/java/org/apache/mailet/base/MailetUtil.java

```diff
@@ -107,4 +109,9 @@ public static boolean getInitParameter(MailetConfig config, String name, boolean
         }
         return defaultValue;
     }
+
+    public static String getInitParameter(MailetConfig config, String name, String defaultValue) {
```

Comment:
Why don't you return Optional so that you can do the `or` at call site instead of having to analyze what means each string parameter ?